### ISSUE TITLE
Use bulk creation with importation methods

### ISSUE THREAD

**Issue by tienne-B** [TabbycatDebate/tabbycat#1199]
For both `importtournament` and web importer, the use of `bulk_create()` could reduce the number of queries being made, with the downside being that the models' `save()` and pre-post- methods are not called, as stated in #1069.

It could be possible to use a pattern like the one I used in #1076 (create lists of objects, then fill foreign keys once bulk created)

**Comment by czlee** [comment]
Yeah, I have to admit, the importer is one of the places where I'd be the most nervous about bypassing `save()` and signals, and it also happens to be one of the places where I'm the least concerned about performance, since you'd naturally expect imports to take a long time.
